content/SCALE/SCALEUIReference/DataProtection/SMARTTestsSCALE.md

```diff
@@ -36,6 +36,10 @@ To quickly test one or more disk for errors, select the disks you want to test a
 
 Next, select the test **Type**.
 Test types differ based on the drive connection, ATA or SCSI:
+
+{{< hint info >}}
+Note: We currently do not support manual S.M.A.R.T. tests on NVMe devices.
```

Comment:
using the hint boxes removes the need to include Note: in the sentence, suggest the following:
```suggestion
Manual S.M.A.R.T. tests on NVMe devices is currently not supported.
```